src/components/dashboard/Pagination.tsx

```
@@ -0,0 +1,66 @@
+import CustomText from '../common/CustomText';
+import {
+  ActivityIndicator,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {Theme} from '../../constants/Types';
+import Icon from '../../assets/Icon';
+
+interface PaginationButtonProps {
+  onLoadMore?: () => void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)